packages/language-server/src/completion-items.ts

```diff
@@ -27,6 +35,7 @@ import {
   typeToString
 } from "@ui5-language-assistant/logic-utils";
 import { getNodeDocumentation } from "./documentation";
+import { Position } from "vscode-languageserver-types";
```

Comment:
import order

Reply:
Moved

Reply:
I still see a relative local import followed by an external import

Reply:
Will move it in another PR